src/components/to-do-list/inputs/Input.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import List from "../lists/List";
+import './input.css'
+
+class Input extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: '',
+            list: []
+        };
+    }
+
+    inputChange = (e) => {
+        this.setState({value: e.target.value});
+    }
+
+    inputSubmit = (e) => {
+        e.preventDefault();
+        this.setState(prevState => ({
+            list: [...prevState.list, this.state.value]
+          }));
+    }
+
+    render() {
+        return (
+            <div>
+                <form onSubmit={this.inputSubmit}>
+                    <input type='text' value={this.state.value} onChange={this.inputChange}></input>
+                    <input type="submit" value="Add" />
```

Comment:
convert to button, set click handler here, instead of form submit
```suggestion
                    <button type="submit" value="Add" onClick={this.inputSubmit} />
```